### ISSUE TITLE
VSCodium: hash mismatch in fixed-output derivation

### ISSUE THREAD

**Issue by ratsclub** [NixOS/nixpkgs#138324]
### Describe the bug

VSCodium has an incorrect sha256 on `x86_64-linux`.

https://github.com/NixOS/nixpkgs/commit/4828eb7ae4f1105dcebf7245f453c607b9c77c1d

### Steps To Reproduce

1. Download VSCodium from the `nixos-unstable` channel.

### Expected behavior

VSCodium should not have a hash mismatch.

### Additional context

```
error: hash mismatch in fixed-output derivation '/nix/store/bxhgf747vapivf2bbqhrxq72g0x0x54h-VSCodium-linux-x64-1.60.1.tar.gz.drv':
         specified: sha256-zCK8pDCw9YGDN+QpRsNZK5vgq6d3cFAeeiE4Zbce/NM=
            got:    sha256-CY7kWvJpyIUylWo4gRfJ+7PioWZ7yxqoNgS4vf7l5g8=
error: 1 dependencies of derivation '/nix/store/8bmhscysqcp5191fzq6gvq61yd5c6jjx-vscodium-1.60.1.drv' failed to build
error: 1 dependencies of derivation '/nix/store/dai2s94mx8wlsg2msqifq198ml3hxk4v-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/l36vpzaqmfqa480zxfndir7255zs337r-home-manager-generation.drv' failed to build
```

### Notify maintainers

@Synthetica9 @turion

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.13.0-7614-generic, Pop!_OS, 21.04`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210908_3c56f62`
 - channels(victor): `"home-manager, nixpkgs-21.11pre314682.65b70fbe4c3"`
 - nixpkgs: `/home/victor/.nix-defexpr/channels/nixpkgs`
```

**Comment by pshirshov** [comment]
Possibly they replaced the file on the server? Could you submit a P/R fixing the hash?

**Comment by figsoda** [comment]
fixed on master: #138161

**Comment by Synthetica9** [comment]
Yeah, this will fix itself with the next channel tick of unstable.

In the meantime, you could do the following if you _really_ need to update your system right now:

```nix
let 
  master = import (builtins.fetchtarball "httsp://github.com/nixos/nixpkgs") { };
in
{
  enviornment.systemPackages = [
    master.vscodium
  ];
}
``` 

See also: https://github.com/VSCodium/vscodium/issues/847
